nodejs/week1/meals/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+
+app.get("/cheap-meals", (req, res) => {
```

Comment:
Again this route has already been registered in index.js, you only need to export the function that handles the request and returns a response (req, res) => {....}